face-db-search/README.md

```diff
@@ -1,382 +1 @@
-```
-
-Run the following command to search images and display them in a webpage.
-
-```bash
-python make_html.py
-```
-
-<details>
-<summary>Click here to see result of old API</summary>
-
-<p align="center">
-  <img src=".github/query-demo.png?raw=true" alt="query flow console output">
-</p>
-
-</details> 
-
-<img src=".github/face-demo.jpg" alt="query flow console output">
-</p>
-Note : We have only used 500 images with only one or two images of each person. Hence the results are only as good as the database you have. It returns the closest matching face in the dataset.
-
-<br /><br />
-
-Congratulations! Now you have an image search engine working at hand. We won't go into details of the Pods' YAML files because they are quite self explained. If you feel a bit lost when reading the YAML files, please check out the [bert-based semantic search demo](https://github.com/jina-ai/examples/tree/master/urbandict-search#dive-into-the-pods).
-
-## Next Steps
-
-- Write your own executors.
-- Check out the docker images at the Jina hub.
-
-## Documentation 
-
-<a href="https://docs.jina.ai/">
-<img align="right" width="350px" src="https://github.com/jina-ai/jina/blob/master/.github/jina-docs.png" />
-</a>
-
-The best way to learn Jina in depth is to read our documentation. Documentation is built on every push, merge, and release event of the master branch. You can find more details about the following topics in our documentation.
-
-- [Jina command line interface arguments explained](https://docs.jina.ai/chapters/cli/index.html)
-- [Jina Python API interface](https://docs.jina.ai/api/jina.html)
-- [Jina YAML syntax for executor, driver and flow](https://docs.jina.ai/chapters/yaml/yaml.html)
-- [Jina Protobuf schema](https://docs.jina.ai/chapters/proto/index.html)
-- [Environment variables used in Jina](https://docs.jina.ai/chapters/envs.html)
-- ... [and more](https://docs.jina.ai/index.html)
-
-## Community
-
-- [Slack channel](https://join.slack.com/t/jina-ai/shared_invite/zt-dkl7x8p0-rVCv~3Fdc3~Dpwx7T7XG8w) - a communication platform for developers to discuss Jina
-- [Community newsletter](mailto:newsletter+subscribe@jina.ai) - subscribe to the latest update, release and event news of Jina
-- [LinkedIn](https://www.linkedin.com/company/jinaai/) - get to know Jina AI as a company and find job opportunities
-- [![Twitter Follow](https://img.shields.io/twitter/follow/JinaAI_?label=Follow%20%40JinaAI_&style=social)](https://twitter.com/JinaAI_) - follow us and interact with us using hashtag `#JinaSearch`  
-- [Company](https://jina.ai) - know more about our company, we are fully committed to open-source!
-
-
-
-## License
-
-Copyright (c) 2020 Jina AI Limited. All rights reserved.
-
-Jina is licensed under the Apache License, Version 2.0. See [LICENSE](https://github.com/jina-ai/jina/blob/master/LICENSE) for the full license text.
+For tutorial, please refer to [here](https://github.com/jina-ai/examples/tree/master/flower-search##similar-examples).
```

Comment:
For tutorial, please refer [here](https://github.com/jina-ai/examples/tree/master/flower-search##similar-examples).

Reply:
avoid the `to here`